core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/Stain.java

```diff
@@ -0,0 +1,42 @@
+package uk.ac.bbsrc.tgac.miso.core.data;
+
+import java.io.Serializable;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "Stain")
+
+public class Stain implements Serializable {
+  private static final long serialVersionUID = 1L;
+  private String stainGroup;
```

Comment:
Yes, but since we either retrieve all the stains or a stain by ID, creating a separate table for stain groups doesn't serve any purpose. The groups are going to be institute specific, so creating an enum is definitely unhelpful. The idea is that for all the antibody stains, we can set the group to be IHC so that they either appear on the same submenu or at least sort together in the drop down.